libs/api-client-bear/src/gooddata-node.ts

```diff
@@ -1,17 +1,22 @@
-// (C) 2007-2020 GoodData Corporation
+// (C) 2007-2022 GoodData Corporation
 
 import fetchCookie from "fetch-cookie";
 import nodeFetch from "node-fetch";
 
 import { factory, SDK } from "./gooddata";
 
+// AbstractFactory always reinitializes fetch-cookie. This way, when factory initialization
+// happens between requests, new instance of fetch-cookie and uderlying cookie jar
+// will be used.
+const abstractFactoryNode = (config?: any) => factory(fetchCookie(nodeFetch as any))(config);
+
 const factoryNode = factory(fetchCookie(nodeFetch as any));
```

Comment:
Can we define `factoryNode` using the `abstractFactoryNode` to reduce duplication?

Reply:
I tried to tackle this, but the only way is `const factoryNode = abstractFactoryNode`, which would be a breaking change.